Resources/translations/SonataAdminBundle.es.xliff

```diff
@@ -204,7 +204,7 @@
             </trans-unit>
             <trans-unit id="title_batch_confirmation">
                 <source>title_batch_confirmation</source>
-                <target>Confirme la operación</target>
+                <target>Confirme la operación '%action%'</target>
```

Comment:
Yes, you are right, I didn't think about it.
Double quotes are more common in spanish.